source/includes/_admin.md

```diff
@@ -76,18 +76,7 @@ adminApi.trigger_invoice_generation_for_parked_accounts('test', api_key, api_sec
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-< Content-Type: application/json
-```
-```java
-no content
-```
-```ruby
-no content
-```
-```python
+```json
 no content
 ```
 
```

Comment:
Instead of just adding `no content` to the response, wondering if we should also add the headers as is done [currently](https://killbill.github.io/slate/?shell#admin-trigger-an-invoice-generation-for-all-parked-accounts):
<img width="402" alt="image" src="https://user-images.githubusercontent.com/85998496/236110324-33f0e092-6848-438a-a8c1-2c60c45bf1b3.png">
Any thoughts @pierre ?

Reply:
I don't think it adds much information to be honest. Customers will very rarely seem or interact with them.